import_export/formats/base_formats.py

```diff
@@ -82,7 +82,7 @@ def get_title(self):
         return self.get_format().title
 
     def create_dataset(self, in_stream, **kwargs):
-        return tablib.import_set(in_stream, format=self.get_title())
+        return tablib.import_set(in_stream, format=self.get_title(), **kwargs)
```

Comment:
Please can you add a unit test to verify that `import_set()` is now called with kwargs.